src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -1,27 +1,23 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Net;
-using System.Runtime.Serialization.Formatters.Binary;
-using System.Text;
-using System.Threading.Tasks;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using Orleans.Runtime;
 
 namespace Orleans.Serialization
 {
-    internal class OrleansJsonSerializer : IExternalSerializer
+    public class OrleansJsonSerializer : IExternalSerializer
     {
-        public static JsonSerializerSettings settings;
+        public static JsonSerializerSettings SerializerSettings { get; set; }
```

Comment:
Should / could this be `get; private set` ?

Are there any situations where someone would want to substitute a whole new SerializerSettings blob rather than just tweak specific sub-properties and/or add extra Converters?